_posts/2017-07-26-a-web-animations-deep-dive-with-angular.md

```diff
@@ -64,13 +64,13 @@ The point is, a "system" or some element on the page can have multiple states. I
+Nowadays, with modern browsers in mind, we have many technologies at hand to animate our UI including CSS3 or JavaScript. The power of CSS animations (transitions or keyframes) is that they allow animating most HTML elements without using JavaScript. CSS animations are also quite fast and allow hardware acceleration. However, there are some limitations to this approach. For instance, you can't animate an element along a certain path, use physics-based motion or animate the scroll position with CSS alone. While CSS animations are pretty good for simple transitions between states (e.g. hover-effects), JavaScript-based animations provide far more flexibility.
 
 As a matter of fact, we can take advantage of the same hardware acceleration in JavaSript too. It's as easy as setting a CSS property with a 3D characteristic, such as `translate3d()` or `matrix3d()`. This will push the particular element onto another layer which is then processed by the GPU. The GPU itself is highly optimized for moving pixels making it much for effective for animations compared to the CPU. For more information, check out this great article by Paul Lewis and Paul Irish on [high performance animations](https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/).
 
 CSS animations do not require any 3rd party libraries. However, there are some tools that make your life much easier, for example libraries that provide pre-defined keyframe animations like [Animate.css](https://github.com/daneden/animate.css).
 
-Looking at JavaScript, we can either use vanilla JavaScript or jQuery to animate our UI. Maintaining vanilla JavaScript animations and manually setting up elements on the stage could be quite a hassle. That's one reason why many of us switched to jQuery at some point. jQuery makes it easy to query an element on the page. Then, in order to add motion, all we do is to call `.animate()` and specify properties (e.g. opacity or transform) we'd like to animate. Here is how we'd move a `div` to the right by `200px` animating its `left` property:
+Looking at JavaScript, we can either use vanilla JavaScript or jQuery to animate our UI. Maintaining vanilla JavaScript animations and manually setting up elements on the stage could be quite a hassle. That's one reason why many of us switched to jQuery at some point. jQuery makes it easy to query an element on the page. Then, in order to add motion, all we do is to call `.animate()` and specify the properties (e.g. opacity or transform) we'd like to animate. Here is how we'd move a `div` to the right by `200px` animating its `left` property:
```

Comment:
I've changed `all we do is to call `.animate()` and specify properties ` to `all we do is to call `.animate()` and specify the properties `